apps/website/docs/pipelines/release-azure-webapp.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 50
+---
+
+# Deploying a Node.js Application to Azure
+
+[Release Azure Webapp](https://github.com/pagopa/dx/blob/main/.github/workflows/release-azure-webapp-v1.yaml)
+is a template for workflows designed for building and deploying **Node.js** web
+applications (including Next.js) to Azure App Services or Azure Function Apps.
+
+## How It Works
+
+**Build and Deployment Overview:**
+
+- It installs the dependencies using `pnpm` or `yarn` and builds the application
+  using `turbo build` command
+- The build output is packaged into a zip artifact that contains the compiled
+  application code and all necessary dependencies in a `node_modules` directory.
+- The artifact is deployed to the specific Azure Web App using a staging slot
```

Comment:
```suggestion
- The artifact is deployed to the specific Azure Web App/Function App using a staging slot
```

Reply:
as decided a few hours ago, I will rename all "web app" references to "appsvc"